exercises/series/src/test/java/SeriesTest.java

```diff
@@ -17,145 +17,117 @@
     public ExpectedException expectedException = ExpectedException.none();
 
     @Test
-    public void hasDigitsShort() {
-        Series series = new Series("01234");
-        List<Integer> expected = Arrays.asList(0, 1, 2, 3, 4);
-        List<Integer> actual = series.getDigits();
-        assertNotNull(actual);
-        assertFalse(actual.isEmpty());
-        assertEquals(expected, actual);
+    public void slicesOfOneFromOne() {
+		Series series = new Series("1");
+		List<List<Integer>> expected = Arrays.asList(Collections.singletonList(1));
+		List<Integer> actual = series.slice(1);
```

Comment:
From the [reference implementation](https://github.com/exercism/java/blob/master/exercises/series/.meta/src/reference/java/Series.java) it looks like this method is called `slices` not `slice` which is why the build is failing 🙂